README.rst

```diff
@@ -117,9 +128,10 @@ The format of a `batch_scoring.ini` file is as follows::
 
 Usage Notes
 -----------
-  * If the script detects that a previous script was interrupted in mid-execution, it will prompt whether to resume that execution.
-  * If no interrupted script was detected or if you indicate not to resume the previous execution, the script checks to see if the specified output file exists. If yes, the script prompts to confirm before overwriting this file.
-  * The logs from each batch_scoring run are stored in the current working. All users will see a `datarobot_batch_scoring_main.log` log file. Windows users will see two additional log file, `datarobot_batch_scoring_batcher.log` and `datarobot_batch_scoring_writer.log`.
+
+* If the script detects that a previous script was interrupted in mid-execution, it will prompt whether to resume that execution.
+* If no interrupted script was detected or if you indicate not to resume the previous execution, the script checks to see if the specified output file exists. If yes, the script prompts to confirm before overwriting this file.
+* The logs from each `batch_scoring` and `batch_scoring_sse` run are stored in the current working directory. All users see a `datarobot_batch_scoring_main.log` log file. Windows users see two additional log files, `datarobot_batch_scoring_batcher.log` and `datarobot_batch_scoring_writer.log`.
```

Comment:
It seems (from the "View" view that the single ` (e.g., `batch_scoring`)does not create what you are intending. All other references to the command you sed double (e.g., ``batch_scoring``)